RELEASE.md

```diff
@@ -1,42 +1,48 @@
 # Making a Release of Notebook
 
-## Start from a fresh git checkout and conda environment
+## Using `jupyter_releaser`
 
-### Set the release branch
+The recommended way to make a release is to use [`jupyter_releaser`](https://github.com/jupyter-server/jupyter_releaser#checklist-for-adoption).
+
+## Manual Release Process
+
+### Start from a fresh git checkout and conda environment
+
+#### Set the release branch
 
 ```bash
 export release_branch=master
 ```
 
-### Create the git checkout
+#### Create the git checkout
 
 ```bash
 git clone git@github.com:jupyter/notebook.git
 cd notebook
 git checkout ${release_banch}
 ```
 
-### Create and activate the conda environment
+#### Create and activate the conda environment
 
 ```bash
 conda create -n notebook-release -c conda-forge jupyter
 conda activate notebook-release
 ```
 
-## Perform a local dev install
+### Perform a local dev install
 
 ```bash
 pip install -ve .
 ```
 
-## Install release dependencies
+### Install release dependencies
 
 ```bash
-conda install -c conda-forge nodejs babel twine
+conda install -c conda-forge nodejs babel twine build
 npm install -g po2json
 ```
 
-## Update the version
+### Update the version
```

Comment:
Should this include `vim pyproject.toml`?  (Same for the 'Change back to dev version' below)

Reply:
No, the version source of truth is in `_version.py` and is read in by `setup.cfg` by value.